README.md

```diff
@@ -68,6 +68,11 @@ A curated list of awesome companies that offer their tools and services for free
 - [VersionEye](https://www.versioneye.com/) - Notification system for software packages.
 - [WhiteSource](https://www.whitesourcesoftware.com/) - Open source security and license management.
 
+## Exception Reporting
+
+- [BugSnag](https://www.bugsnag.com/) `requires-approval` - Exception tracking and reporting.
+- [Honeybadger](https://www.honeybadger.io) `requires-approval` `non-commercial` - Exception reporting and uptime monitoring.
```

Comment:
I like the `non-commercial` tag! Would you mind adding a little explanation for it in the **Tags** section at the bottom of the README, please? Don't worry about formatting, I can clean that up after, just a little blurb explaining what `non-commercial` means.

Reply:
Also, Honeybadger already exists under the **Monitoring** category, but I'm not against creating a separate **Exception Reporting** category for these types of tools. Rather than duplicate it, would you please remove it from the **Monitoring** category below?

Reply:
Oh, sorry, I meant to, just forgot to remove it after moving it :)